lib/tasks/pa11y.js

```diff
@@ -28,21 +28,20 @@ const pa11yTest = async function (config) {
 		]
 	});
 
-	// Process and return the results
-	require('colors');
-	const errors = results.issues.filter(result => result.type === 'error')
-		.map(result => '\n' +
-			' • Error: ' + result.message.white +
-			'\n' +
-			('   ├── ' + result.code +
-				'\n' +
-				'   ├── ' + result.selector.replace(/\s+/g, ' ') +
-				'\n' +
-				'   └── ' + result.context.replace(/\s+/g, ' ')).grey
-		);
-
 	if (results.issues.some(result => result.type === 'error')) {
-		throw new Error(errors.join('\n') + '\nFailed Pa11y tests');
+		// Process and return the results
+		require('colors');
```

Comment:
oh what does this `require('colors');` do?

Reply:
It adds lots of methods to the String prototype 🙈 Let's fix this to use `colors/safe` instead